doc/using_pytorch.rst

```diff
@@ -303,7 +314,8 @@ It loads the model parameters from a ``model.pth`` file in the SageMaker model d
 
```

Comment:
What do you think about changing the existing examples to use `model.pt` as well? Or are you against it because people have to implement their own model_fn without EI? I'm worried people will get confused

Reply:
As discussed offline, changing all ``.pth`` to ``.pt`` will make us change all previous test scripts, docs and notebook examples.